packages/core/ui/frame/index.android.ts

```diff
@@ -1301,13 +1301,18 @@ class ActivityCallbacksImplementation implements AndroidActivityCallbacks {
 		if (!rootView) {
 			const mainEntry = application.getMainEntry();
 			const intent = activity.getIntent();
+			let ensureNoRootView = false;
 
 			if (fireLaunchEvent) {
 				// entry point for Angular and Vue frameworks
 				rootView = notifyLaunch(intent, <any>savedInstanceState, null);
+				ensureNoRootView = rootView === null;
 			}
 
 			if (!rootView) {
+				if(ensureNoRootView) {
```

Comment:
```suggestion
				if(rootView === null) {
				  // root view is explicitly set to null - bail
```
The `ensureNoRootView` variable seems redundant (and confusing a bit)

Reply:
It's mainly for consistency. In iOS if you try to `resetRootView(null)` it'll throw, so this is to ensure that rootView can only be null during the launch event, and that specific function is called both on launch (`fireLaunchEvent = true`) and on `resetRootView`

Reply:
Oh ok - thanks for clarifying! I still think the name could be changed to something more "self-explanatory"

Reply:
Went with this more clear name and added comment:
```
// useful for integrations that would like to set rootView asynchronously after app launch
let shouldRootViewBeEmpty = false;
```